services/metadactyl-clj/project.clj

```diff
@@ -44,4 +44,5 @@
   :repositories [["sonatype-nexus-snapshots"
                   {:url "https://oss.sonatype.org/content/repositories/snapshots"}]]
   :deploy-repositories [["sonatype-nexus-staging"
-                         {:url "https://oss.sonatype.org/service/local/staging/deploy/maven2/"}]])
+                         {:url "https://oss.sonatype.org/service/local/staging/deploy/maven2/"}]]
```

Comment:
Heh, I just realized that the sonatype snapshot repository definitions can go away. I created a Jira issue for it.